canonical-web-and-design/charmhub.io
# Add missing categories

## Done

Add missing categories

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Make sure all categories are here
